CMakeLists.txt

```diff
@@ -1575,6 +1596,23 @@ if(HPX_WITH_APEX)
   endif()
 endif()
 
+if(HPX_WITH_HPXMP)
+  set(_hpxmp_no_update)
+  if(HPX_WITH_HPXMP_NO_UPDATE)
+    set(_hpxmp_no_update NO_UPDATE)
+  endif()
+
+  # handle hpxMP library
+  include(GitExternal)
+  git_external(hpxmp
+    https://github.com/STEllAR-GROUP/hpxMP
```

Comment:
Shouldn't this be https://github.com/STEllAR-GROUP/hpxMP.git ? Looks like there is a typo where .git is missing in the end.

Reply:
@justwagle yes, thanks!